src/common/components/ui/formik/MultiSelectField.tsx

```diff
@@ -29,11 +29,11 @@ const MultiSelectField: React.FC<MultiSelectFieldProps> = ({
   ...props
 }) => {
   const [isOpen, setOpen] = React.useState(false);
-  const [field, { touched, error }, { setValue }] = useField(props.name);
+  const [field, { touched, error }, { setValue }] = useField<string[]>(props.name);
   const fieldId = getFieldId(props.name, 'multiinput', idPostfix);
   const isValid = !(touched && error);
   const errorMessage = !isValid ? error : '';
-  const hText = getHelperText ? getHelperText(field.value) : helperText;
+  const hText = getHelperText ? getHelperText(field.value?.join(',')) : helperText;
```

Comment:
`field.value?.join(',')` will evaluate to `undefined` when there is no `value`. Maybe appending `?? ''`  will help?  
Also what I find interesting is w/o peeking into `getHelperText` definition, how much responsibility this function has.
Looks like it's fine accepting undefined values, and if its type is `field.value: string[]` then it should know what to do with that array already. So perhaps there is no need to call `.join(',')`???

Reply:
done